### DISCUSSION TITLE
Sigma/ElastAlert Configuration

### DISCUSSION THREAD

**Discussion by chrismagowan** [Security-Onion-Solutions/securityonion#2507] [Unsupported Versions] [answered]
I've enabled Play #337 (Suspicious Certutil Command) to test out sigma alerts. However, I'm not seeing the alert trigger. I noticed the ElastAlert config will return the expected results if I remove ".security" and manually run the query, but returns no results if I run the query as is. The results I get are sysmon events (event ID 1/process creation). The logs are being shipped from a windows 10 host via winlogbeat and sysmon. SO deployment is a single node/eval (2.3.21), and so-status returns no errors.

Exact sigma code being used is from [https://github.com/Neo23x0/sigma/blob/master/rules/windows/process_creation/win_susp_certutil_command.yml](url)

Unmodified ElastAlert Config (returns no results):

`filter:`
`- query:`
 `       query_string:`
   `   query: (event.code.security:"1" AND winlog.channel.security:"Microsoft\-Windows\-Sysmon\/Operational" AND process.command_line.security:("*\ \-decode\ *" OR "*\ \/decode\ *" OR "*\ \-decodehex\ *" OR "*\ \/decodehex\ *" OR "*\ \-urlcache\ *" OR "*\ \/urlcache\ *" OR "*\ \-verifyctl\ *" OR "*\ \/verifyctl\ *" OR "*\ \-encode\ *" OR "*\ \/encode\ *" OR "*certutil*\ \-URL*" OR "*certutil*\ \/URL*" OR "*certutil*\ \-ping*" OR "*certutil*\ \/ping*"))`
`index: '*:so-*'`
`name: Suspicious Certutil Command - d3c930f63`
`priority: 2`
`realert:`
`  minutes: 0`
`type: any`

**Comment by defensivedepth** [comment, marked as answer]
Thanks for reporting this! We have an issue open to track this - we are currently working on a fix.

https://github.com/Security-Onion-Solutions/securityonion/issues/2460#issuecomment-752108652

Thanks!
